homeassistant/components/hunterdouglas_powerview/select.py

```diff
@@ -67,26 +62,22 @@ async def async_setup_entry(
 
     pv_entry: PowerviewEntryData = hass.data[DOMAIN][entry.entry_id]
 
-    entities = []
-    for raw_shade in pv_entry.shade_data.values():
-        shade: BaseShade = PvShade(raw_shade, pv_entry.api)
-        if SHADE_BATTERY_LEVEL not in shade.raw_data:
-            continue
```

Comment:
We want to keep the continue pattern where possible to avoid creating long indent trees

Reply:
> Please reverse conditionals and continue to avoid long indents

this is effectively added back in by this change https://github.com/home-assistant/core/pull/108066/files/2fc81acb5017172a8c48babcfb67b130568df44a#r1476621073

``shade.has_battery_info()`` replaces this statement and returns the required based on the hub version